livekit_egress.proto

```diff
@@ -35,11 +35,6 @@ message RoomCompositeEgressRequest {
   bool audio_only = 3;          // (default false)
   bool video_only = 4;          // (default false)
   string custom_base_url = 5;   // template base url (default https://recorder.livekit.io)
-  oneof output {                // deprecated (use _output fields)
```

Comment:
ok, we cannot deprecate this yet. this is a major change.

especially for Cloud customers, we need to ensure everyone's migrated off before pulling the trigger.

Reply:
I can leave the fields in for now and just remove the rpc system